# Define the goal
sensor_msgs/PointCloud2 pointcloud
---
# Define the result
geometry_msgs/Point grasping_point
---
# Define a feedback message
float32 percent_complete